src/rgw/setup/rgw.py

```diff
@@ -126,6 +126,14 @@ def config(conf: MappedConf):
         # Before user creation,Verify backend store value=motr in rgw config file.
         Rgw._verify_backend_store_value(conf)
 
+        # Create motr trace directory for collecting m0trace files
+        # in case admin user creation issue during mini-provisioner execution.
+        Log.info(f'Creating motr trace directory for collecting m0trace files..')
+        log_path = Rgw._get_log_dir_path(conf)
+        motr_trace_dir = os.path.join(log_path, 'motr_trace_files')
+        os.makedirs(motr_trace_dir, exist_ok=True)
+        os.environ['M0_TRACE_DIR'] = motr_trace_dir
```

Comment:
setting the env variable will tell libMotr to use this dir for dumping m0trace files?

Reply:
yes.thats correct.